src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -56,5 +56,40 @@ DataStore.configure({
 });
 ```
 
+</Block>
+<Block name="Expo">
+
+Start with the [Expo](https://docs.expo.dev/):
+
+```bash
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npx amplify-app@latest
+npm install @react-native-community/netinfo @react-native-async-storage/async-storage
+```
+
+**Use SQLite for enhanced performance (optional)**
+
+Expo built apps can now use SQLite with DataStore. SQLite offers considerable performance advantages over the default "AsyncStorage" database.
+
+To enable SQLite with DataStore for enhanced local database performance, follow the steps below:
+
+```sh
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npm install aws-amplify @aws-amplify/datastore-storage-adapter expo-sqlite aws-amplify-react-native amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
```

Comment:
Is it essential to install these to get started?
`aws-amplify-react-native`, `amazon-cognito-identity-js` , `@react-native-picker/picker`

Reply:
Good point, `aws-amplify-react-native` and `@react-native-picker/picker` will not be needed. However, `amazon-cognito-identity-js` is needed as aws-amplify depends on it. I'll remove those additional packages.